vimeo-networking/src/main/java/com/vimeo/networking/Configuration.java

```diff
@@ -332,24 +309,16 @@ public Builder(String clientId, String clientSecret, String scope,
          * @param accountStore  (Optional, Recommended) An implementation that can be used to interface with Android's
          *                      <a href="https://goo.gl/QZ7rm">Account Manager</a>
          */
-        @Deprecated
-        public Builder(@Nullable String baseURLString,
-                       String clientId,
-                       String clientSecret,
-                       String scope,
+        public Builder(@NotNull String clientId,
```

Comment:
We had marked the Builder that allowed the base URL to be set as `Deprecated` so that consumers except for us would not use it. The reasoning is that all consumers should always use `api.vimeo.com` as the base url and would never have any reason to change it. We were the only ones that should have been passing the base url in. What I found from personal experience was that I rarely ever passed anything other than `api.vimeo.com` in. And in order to pass anything else in, I had to make a change to the code anyway. So why not just change it here? 

Also what I found, was that we had some bad logic flow. Even when passing anything other than `api.vimeo.com` in, it was always overwritten after we received a response to `/configs`. This caused this to not work as intended. I felt that it was too confusing to follow so I tried to simplify that, which is why I removed the ability to set the base url when creating a configuration object.